core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -391,14 +391,24 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
         // exec is given.
         val parameters = content.exec map {
             case seq: SequenceExec => Parameters("_actions", JsArray(seq.components map { JsString(_) }))
-            case _                 => content.parameters getOrElse Parameters()
+            case _ => content.parameters getOrElse {
+                action.exec match {
+                    case seq: SequenceExec => Parameters()
+                    case _                 => action.parameters
+                }
+            }
+        } getOrElse {
+            action.exec match {
+                case seq: SequenceExec => action.parameters // discard content.parameters
+                case _                 => content.parameters getOrElse action.parameters
+            }
```

Comment:
I'dd suggest to add a few comments to see with one few, in which edge case which Parameters are set and why.